controls/cf_serverd.cf

```diff
@@ -45,12 +45,25 @@ body server control
       cfruncommand => "$(sys.cf_agent) -I -D cfruncommand -f \"$(sys.update_policy_path)\"  &
                        $(sys.cf_agent) -I -D cfruncommand";
 
-    !windows::
-      # last single quote in cfruncommand is left open, so that
-      # arguments (like -K and --remote-bundles) are properly appended.
-      cfruncommand => "$(def.cf_runagent_shell) -c \'
-                           $(sys.cf_agent) -I -D cfruncommand -f $(sys.update_policy_path)  ;
-                           $(sys.cf_agent) -I -D cfruncommand";
+    !windows.!cfengine_3_7::
+
+      # in cfengine 3.10.0 and later last single quote in cfruncommand is left
+      # open, so that arguments (like -K and --remote-bundles) are properly
+      # appended.
+
+        cfruncommand => "$(def.cf_runagent_shell) -c \'
+                           $(sys.cf_agent) -I --define cfruncommand -f $(sys.update_policy_path)  ;
+                           $(sys.cf_agent) -I --define cfruncommand";
```

Comment:
So you only changed the style from `-D` to `--define` ? I don't like the new style :-p

Reply:
I can see in the serverd code that `-D` is treated specially to check if "role activation" is permitted. I can't see `--define` there, so I would suggest leaving it `-D`.

Reply:
Bah, fine. Yeah, I much prefer the long options. It's way easier to read than knowing all the abbreviations.